tests/unit/python/foglamp/services/core/api/test_service.py

```diff
@@ -181,16 +178,13 @@ def q_result(*arg):
         with patch.object(connect, 'get_storage', return_value=storage_client_mock):
             with patch.object(storage_client_mock, 'query_tbl_with_payload', side_effect=q_result):
                 with patch.object(storage_client_mock, 'insert_into_tbl', return_value={'rows_affected': 1, "response": "inserted"}) as insert_table_patch:
-                    with patch.object(c_mgr, 'create_category', return_value=async_mock()) as patch_create_cat:
+                    with patch.object(c_mgr, 'create_category', return_value=None) as patch_create_cat:
                         resp = await client.post('/foglamp/service', data=json.dumps(data))
                         assert 400 == resp.status
                         assert 'A schedule with that name already exists' == resp.reason
-                    patch_create_cat.assert_called_once_with(category_name=data['name'], category_description=description, category_value=val, keep_original_items=True)
-                args, kwargs = insert_table_patch.call_args
-                assert 'scheduled_processes' == args[0]
-                p = json.loads(args[1])
-                assert {'name': 'furnace4', 'script': '["services/north"]'} == p
+                    assert 0 == patch_create_cat.call_count
 
+    @pytest.mark.skip(reason="TODO: Mock loading a plugin module")
```

Comment:
Please file a JIRA for this; and mention in reason `reason="TODO:  FOGL-xxx Mock loading a plugin module"`